src/app-gocardless/gocardless-node.types.ts

```diff
@@ -356,7 +356,11 @@ export type Transaction = {
   /**
    * Account reference, conditional
    */
-  creditorAccount?: string;
+  creditorAccount?:
+    | string
+    | {
+        iban?: string;
+      };
```

Comment:
As it happens, this can just be an object. It was wrongly typed before.

```suggestion
  creditorAccount?: {
    iban: string;
  };
```

Reply:
@matt-fidd I did this, but it seems to fail to build, as there are cases/banks where `creditorAccount` really is just a string. Shall we just keep it as the original optional string or object?

Reply:
Ah okay, leave it as you've done it for now and we can fix in other places in another PR

Reply:
Alright :) Thanks for the quick feedback.